docs/fiddles/features/progress-bar/main.js

```diff
@@ -3,19 +3,28 @@ const { app, BrowserWindow } = require('electron')
 function createWindow () {
   const win = new BrowserWindow({
     width: 800,
-    height: 600,
-    webPreferences: {
-      nodeIntegration: true
-    }
+    height: 600
   })
 
   win.loadFile('index.html')
-  win.setProgressBar(0.5)
-}
 
+  const INCREMENT = 0.02
+  const INTERVAL_DELAY = 100 // ms
+  let c = 0
+  const interval = setInterval(() => {
```

Comment:
Idea: should we chain `setTimeout(...)` calls here, instead of using the interval?

Reply:
I think `setInterval` is right for doing something _every x milliseconds_. The same can be accomplished with a loop that uses a `setTimeout` at the end, but then the interval becomes 1) the amount of time it takes to process the logic in the body of the loop plus 2) the timeout value. With an interval, it starts the block every x milliseconds regardless of how much time the logic takes.